tests/components/axis/test_init.py

```diff
@@ -7,20 +7,17 @@
 from homeassistant.components.axis.const import DOMAIN as AXIS_DOMAIN
 from homeassistant.setup import async_setup_component
 
-from .test_device import setup_axis_integration
-
 
 async def test_setup_no_config(hass):
     """Test setup without configuration."""
     assert await async_setup_component(hass, AXIS_DOMAIN, {})
     assert AXIS_DOMAIN not in hass.data
 
 
-async def test_setup_entry(hass, config_entry):
+async def test_setup_entry(hass, setup_config_entry):
     """Test successful setup of entry."""
-    await setup_axis_integration(hass, config_entry)
     assert len(hass.data[AXIS_DOMAIN]) == 1
-    assert config_entry.entry_id in hass.data[AXIS_DOMAIN]
+    assert setup_config_entry.entry_id in hass.data[AXIS_DOMAIN]
```

Comment:
Assert the config entry state instead of checking things in hass.data.

Reply:
As this PR isn't about changing the tests but the infrastructure is it Ok if I follow up with a new PR regarding the comments?

Reply:
Fixed in https://github.com/home-assistant/core/pull/86648